src/manifest.rs

```diff
@@ -64,17 +64,28 @@ pub(crate) fn get_project_dir_from_manifest_path(
 }
 
 /// A [package.metadata] or [workspace.metadata] table with
-/// `cargo-semver-checks` lint entries stored in `config`
+/// `cargo-semver-checks` config entries stored in `config`
```

Comment:
Let's clarify which of these refer to TOML file sections and which refer to Rust fields:
```suggestion
/// `cargo-semver-checks` config entries stored in the `config` field below.
```